Makefile

```
@@ -716,32 +717,38 @@ $(AUTOHYDRATE_STAMPS):
 # <old-path>" before the compiler ever runs to regenerate the .d. Cheap
 # scan: just stat the first .c prereq the .d declares.
-.PHONY: clean-stale-deps
-clean-stale-deps:
-	$(V1) if [ -d "$(OBJECT_DIR)" ]; then \
-	    find "$(OBJECT_DIR)" -name '*.d' 2>/dev/null | while IFS= read -r dfile; do \
-	        src=$$(awk '{ for (i=1; i<=NF; i++) if ($$i ~ /\.c$$/) { print $$i; exit } }' "$$dfile" 2>/dev/null); \
-	        if [ -n "$$src" ] && [ ! -f "$$src" ]; then \
-	            echo "Removing stale dependency file: $$dfile (source moved: $$src)"; \
-	            $(RM) "$$dfile"; \
+.PHONY: validate-deps
+validate-deps: | $(OBJECT_DIR)
```

checkmake (0.3.2)

[warning] Target body for "validate-deps" exceeds allowed length of 5 lines (15).

(maxbodylength)